rxandroid/src/test/java/io/reactivex/rxjava3/android/schedulers/HandlerSchedulerTest.java

```diff
@@ -145,7 +144,7 @@ public void directScheduleOnceWithDelayPostsWithDelay() {
         runUiThreadTasks();
         assertEquals(0, counter.get());
 
-        idleMainLooper(1, MINUTES);
+        ShadowLooper.idleMainLooper(1, MINUTES);
```

Comment:
Can you static import this to reduce changes?

Reply:
static import done